docs/content/users/usage/cms-settings.md

```diff
@@ -125,4 +125,4 @@ baseVariants:
     condition: 'applicationContext == "Development/DDEV"'
 ```
 
-See also [TYPO3 Documentation](https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/SiteHandling/BaseVariants.html).
+See also [TYPO3 Documentation](https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/SiteHandling/BaseVariants.html).
```

Comment:
Is there a better way to make this permanent so it will always track properly? Seems like 12.4 will be very fragile very fast.

Reply:
Rendering of main branch didn't work but up again. We can change again to main.